test/api/importcsv/samples.go

```diff
@@ -0,0 +1,67 @@
+package importcsv
+
+import (
+	"github.com/konveyor/tackle2-hub/api"
+)
+
+type TestCase struct {
+	fileName             string
+	ExpectedApplications []api.Application
+	ExpectedDependencies []api.Dependency
+}
+
+var (
+	TestCases = []TestCase{
+		{
+			fileName: "template_application_import.csv",
+			ExpectedApplications: []api.Application{
```

Comment:
So the data in this CSV is supposed to be equivalent to what is being described in the `template_application_import.csv`?

Reply:
> So the data in this CSV is supposed to be equivalent to what is being described in the `template_application_import.csv`?

Ya even I think so because we are checking that if all the entities are being uploaded or not thus have to check them all